daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -80,6 +83,28 @@ func init() {
 	}
 }
 
+func getStreamName(ctx logger.Context) (name string, err error) {
+	if ctx.Config[logStreamKey] != "" {
+		name = ctx.Config[logStreamKey]
+	} else if ctx.Config[logStreamTemplateKey] != "" {
+		var t *template.Template
+		t, err = template.New("t").Parse(ctx.Config[logStreamTemplateKey])
+		if err == nil {
```

Comment:
It's not swallowed (it just has no detrimental effect on the rest of the function, so I don't return early) - see the test on 725 - but I'm happy to add an explicit return for clarities sake.